barretenberg/cpp/scripts/benchmark_client_ivc.sh

```diff
@@ -1,26 +1,35 @@
 #!/usr/bin/env bash
 set -eu
 
-TARGET="client_ivc_bench"
-# Note: to run structured trace version, change "Full" to "FullStructured" here and in analyze script
-FILTER="ClientIVCBench/Full/6$"
-BUILD_DIR=build-op-count-time
+TARGET=${1:-"client_ivc_bench"}
```

Comment:
I'm kind of leaving things defaulted to client_ivc for now (including the name of this script) but this can likely be cleaned up in the near future. We really don't need to bench the old client_ivc anymore but I want to leave it around until we can get consensus on what's going on.